java/README.md

```diff
@@ -90,20 +100,105 @@ Other useful gradle developer commands:
 * `./gradlew :examples:run` to run client examples (make sure you have a running redis on port `6379`)
 * `./gradlew :benchmarks:run` to run performance benchmarks
 
+
+### Setting up the Driver
+
+Refer to https://central.sonatype.com/search?q=glide&namespace=software.amazon.glide for your specific system.
+Once set up, you can run the basic examples.
+
+Gradle:
+- Copy the snippet and paste it in the `build.gradle` dependencies section.
+Example shown below is for `glide-osx-aarch_64`.
+```bash
+dependencies {
+    testImplementation platform('org.junit:junit-bom:5.10.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+    implementation group: 'software.amazon.glide', name: 'glide-osx-aarch_64', version: '0.4.2'
+}
+```
+
+Maven (AARCH_64) specific.
+- **IMPORTANT** must include a `classifier` block. Please use this dependency block instead and add it to the pom.xml file.
+```xml
+<dependency>
+    <groupId>software.amazon.glide</groupId>
+    <artifactId>glide-osx-aarch_64</artifactId>
+    <classifier>osx-aarch_64</classifier>
+    <version>0.4.2</version>
+</dependency>
+```
+
 ## Basic Examples
 
-### Standalone Redis:
+### Standalone Valkey:
 
 ```java
 import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+
+import java.util.concurrent.ExecutionException;
+import static glide.api.models.GlideString.gs;
+
+# Run this code in the Main file. Include InterruptedException and ExecutionException handling.
+
+public static void main(String[] args) throws InterruptedException, ExecutionException {
+
+String host = "localhost";
+Integer port = 6379;
+boolean useSsl = false;
+
+RedisClientConfiguration config =
+        RedisClientConfiguration.builder()
+                .address(NodeAddress.builder().host(host).port(port).build())
+                .useTLS(useSsl)
+                .build();
+
+RedisClient client = RedisClient.CreateClient(config).get();
```

Comment:
`RedisClient` -> `GlideClient`
Renaming is undergo in https://github.com/aws/glide-for-redis/pull/1671